### PR TITLE
dev-util/shards: Introduce new package

### PR DESCRIPTION
**Pull request by longlene**
Dependency manager for the Crystal language

### CODE REVIEW

**Comment by monsieurp** [comment]
@gentoo/proxy-maint

**Comment by monsieurp** [comment]
I made two recommendations. Update accordingly and I reckon this PR is good to go.

File: dev-util/shards/shards-0.5.4.ebuild [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
```
Comment (jlec) [inline dev-util/shards/shards-0.5.4.ebuild]: Why not EAPI=6?

**Comment by jlec** [comment]
Any reason you are adding three version instead of the latest?

**Comment by idella** [comment]
1. I made two recommendations. I do not see them any where.
2. In the past you have submitted by use of git commit. Can you please attempt to submit the final patch by use of repoman commit. It's not obligatory but it is preferred, thnx.

**Comment by gktrk** [comment]
Just a heads up, the metadata.xml is missing proxy-maint as a maintainer

**Comment by idella** [comment]
longlene is already established proxy maintainer

**Comment by gktrk** [comment]
Errr... this one:
```
<maintainer type="project">
	<email>proxy-maint@gentoo.org</email>
	<name>Gentoo Proxy Maintainers Project</name>
</maintainer>
```

**Comment by idella** [comment]
Ah yes now brain in gear, quite correct. metadata.xml is only half done

**Pull request merged**